src/components/StructBuilder.js

```diff
@@ -89,17 +90,17 @@ export const StructBuilder = props => {
           {
             size: { basis: 160, grow: 0 },
             field: 'field',
-            headerRenderer: () => h(HeaderCell, ['Field']),
+            headerRenderer: () => h(HeaderCell, ['Variable']),
             cellRenderer: ({ rowIndex }) => {
-              return h(TextCell, { }, [currentStructType.fields[rowIndex].field_name])
+              return h(TextCell, { style: Utils.inputTypeStyle(currentStructType.fields[rowIndex].field_type) }, [currentStructType.fields[rowIndex].field_name])
```

Comment:
This and L103 were changed to make it consistent with styling of Optional inputs in Inputs table